### ISSUE TITLE
Need an option to have more than 500MB storage

### ISSUE THREAD

**Issue by philip-coats-softbank** [Azure/static-web-apps#557]
**Is your feature request related to a problem? Please describe.**
My site is bigger than 500MB.

**Describe the solution you'd like**
I would like a billing plan that has an option for up to 1.5GB of storage.

**Describe alternatives you've considered**
I can store the assets somewhere else, but that really defeats the point of having a single solution for our website.

**Additional context**

**Comment by XavierGeerinck** [comment]
+1 my blog has a lot of pictures and I am hitting the limit as well :(

**Comment by nitelite** [comment]
If you are just using it for a static website, without authentication or APIs, I would suggest checking if maybe using an Azure Storage account with Static Website enabled might be better. The performance of a Storage account is also better for purely static files. It is quite a bit more simplistic than Static Web Apps though.

**Comment by miwebst** [comment]
You lose a lot of functionality (i.e. routing, global distribution, etc.) in addition to auth and api's if you use a storage account. Of course you can add things like AFD/CDN in front to achieve some of these but that increases cost and complexity.

@philip-coats-softbank, do you have the same situation as Xavier? Lots of media content?

**Comment by philip-coats-softbank** [comment]
yes - lots of media on our website.  the entire point of using an Azure Static Web App is simple setup and deployment.  if i have to build all of that infrastructure myself, then I'm not going to use an Azure Static Web App.  

it's probably using Azure Storage behind the scenes anyway - there is no reason they shouldn't be able to add a higher pricing tier for more storage.

**Comment by lukaszch** [comment]
I fully agree that such a hard limit doesn't make much sense. Please provide billing to let us use as much storage as needed.

We want to deploy our documentation to Azure Static Web Sites, but we might not fit into the limit with our thousands of screenshots.

**Comment by barmang** [comment]
I am facing same issue for our documentation service, there are **multiple small text files and images** which all together is increasing content size beyond 500MB. Since we utilize lot of features coming out of the box from Azure Static Web App (ASWA), we do not want to move away from ASWA. Ideal solution would be to have Azure addressed this limitation and provide ways to scale on demand beyond 500 MB up to few GBs. 

Meanwhile, are there any workarounds to resolve size limitation by tweaking the architecture, **Keeping ASWA at the center** but somehow add storage accounts to pull more content from it ?

**Comment by mustafasadikot** [comment]
We have a similar use case where our product documentation is deployed on ASWA. This hard limit is kind of a deal breaker.
